modules/auth.js

```diff
@@ -23,79 +23,51 @@ var CASAuthentication = require('cas-authentication');
 //Load db to acess user data
 var mongoose = require("mongoose");
 require('../models/user.js');
+var config = require('../config/config.js');
 var User = mongoose.model('User');
 
-adminRights = {
-  NONE: 0,
-  BANDANA: 1,
-  PERMIE: 2,
-  ADMIN: 3
-}
-
 var cas = new CASAuthentication({
   cas_url     : 'https://cas.usherbrooke.ca',
   service_url : 'https://localhost:' + ('8443')
 });
 
-setSessionUserInfo = function (req, callback) {
-  var cip = req.session[cas.session_name];
-  User.findOne({ cip: cip }, function (err, obj) {
-    if (obj) {
-      req.session.userInfo = obj;
-    }
-    
-    callback();
-  });
-}
-
-check = function (req, res, next) {
-  if (!req.session.userInfo) {
-    var cip = req.session[cas.session_name];
-    User.findOne({ cip: cip }, function (err, obj) {
-      if (!obj) {
-        res.render("add_user", { cip: cip });
-      } else {
-        setSessionUserInfo(req, function () {
-          next();
-        });
-      }
-    });
-  } else {
-    next();
-  }
-}
-
 allow = function (rights) {
   return function (req, res, next) {
-    var cip = req.session[cas.session_name];
-    User.findOne({ cip: cip }, function (err, obj) {
-      var found = false;
-      if (obj) {
-        rights.forEach(function (element, index, array) {
-          if (obj.rights == element) {
-            found = true;
-            next();
-          }
+    if (!req.session[cas.session_name]) {
+      res.status('403');
```

Comment:
403: The request was a valid request, but the server is refusing to respond to it. Unlike a 401 Unauthorized response, authenticating will make no difference.
Dans ce cas, le user est supposer être déjà logger et donc ça ne ferait aucune différence.